### PR TITLE
Upgrade manubot & environment on 2018-08-22

### PR DESCRIPTION
**Pull request by dhimmel**
Work in progress.

### CODE REVIEW

**Comment by dhimmel** [comment]
Running the following:

```
manubot cite --render \
  --csl=https://github.com/greenelab/manubot-rootstock/raw/2a4a44a5088e587aa25b9655f93842a264362038/build/assets/style.csl \
  doi:10.7554/eLife.32822 pmid:29424689 pmcid:PMC5832410 url:https://greenelab.github.io/meta-review/ 
```

Produced the following:

1. Sci-Hub provides access to nearly all scholarly literature
Daniel S Himmelstein, Ariel Rodriguez Romero, Jacob G Levernier, Thomas
Anthony Munro, Stephen Reid McLaughlin, Bastian Greshake Tzovaras, Casey
S Greene
eLife (2018-03-01) https://doi.org/ckcj
DOI: 10.7554/elife.32822 · PMID: 29424689 · PMCID: PMC5832410

2. Sci-Hub provides access to nearly all scholarly literature.
Daniel S Himmelstein, Ariel Rodriguez Romero, Jacob G Levernier, Thomas
Anthony Munro, Stephen Reid McLaughlin, Bastian Greshake Tzovaras, Casey
S Greene
eLife (2018-03-01) https://www.ncbi.nlm.nih.gov/pubmed/29424689
DOI: 10.7554/elife.32822 · PMID: 29424689 · PMCID: PMC5832410

3. Sci-Hub provides access to nearly all scholarly literature
Daniel S Himmelstein, Ariel Rodriguez Romero, Jacob G Levernier, Thomas
Anthony Munro, Stephen Reid McLaughlin, Bastian Greshake Tzovaras, Casey
S Greene
eLife (2018) https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5832410/
DOI: 10.7554/elife.32822 · PMID: 29424689 · PMCID: PMC5832410

4. Open collaborative writing with Manubot
Daniel S. Himmelstein, David R. Slochower, Venkat S. Malladi, Casey S.
Greene, Anthony Gitter
(2018-08-03) https://greenelab.github.io/meta-review/

Relates to discussion in https://github.com/greenelab/manubot/issues/45 and https://github.com/greenelab/manubot/pull/49#discussion_r209916658

File: build/assets/style.csl [hunk 1]
```diff
@@ -10,7 +10,7 @@
       <name>Daniel Himmelstein</name>
       <email>daniel.himmelstein@gmail.com</email>
     </author>
-    <updated>2017-09-04T20:54:02+00:00</updated>
+    <updated>2018-08-22T19:08:46+00:00</updated>
     <rights license="https://creativecommons.org/publicdomain/zero/1.0/legalcode">This work is dedicated to the public domain via CC0 1.0</rights>
   </info>
   <macro name="author">
```
Comment (agitter) [inline build/assets/style.csl]: As long as you are editing the style, is it easy to add a default value when there is no author?  I'll open a separate issue if it is out of scope.

See https://greenelab.github.io/meta-review/#ref-HPKoE9m3 for an example reference without an author.  The `title` and `venue` are concatenated without whitespace.

**Comment by dhimmel** [comment]
> As long as you are editing the style, is it easy to add a default value when there is no author? I'll open a separate issue if it is out of scope.

@agitter you possibly could try [manubot.csl.txt](https://github.com/greenelab/manubot-rootstock/files/2316368/manubot.csl.txt)... I added a conditional statement if `author` is not present, but I don't think it necessarily will work.

Anyways, I am moving and shutting down my systems... so you'll have to proceed without me.

**Comment by dhimmel** [comment]
Not going to get a chance to do more CSL work before returning. @agitter consider merging this. Perhaps would be good to have a bit more testing of the CSL changes, perhaps with the meta-review bibliography. Up to you whether to wait or proceed

**Comment by agitter** [comment]
@dhimmel I'll review this once more and squash merge once I approve.  I'll create a new issue regarding CSL, or add to our existing issue.

Can you please remind me what I need to do for the squash commit message so that the issue doesn't appear in other downstream repositories?  Do I include the full URL `https://github.com/greenelab/manubot-rootstock/pull/132`?

**Comment by dhimmel** [comment]
Just make sure not to have any relative github references... So yes use the full PR url in the message. The problem we want to avoid is this commit incorrectly linking to another PR in another repository

**Pull request merged**